session/session.go

```diff
@@ -413,11 +414,16 @@ func (s *session) CommitTxn(ctx context.Context) error {
 	}
 
 	stmt := executor.ExecStmt{
-		Text:      "commit",
+		Text:      "commitTxn",
 		Ctx:       s,
 		StartTime: time.Now(),
 	}
+	var commitDetail *execdetails.CommitDetails
+	ctx = context.WithValue(ctx, execdetails.CommitDetailCtxKey, &commitDetail)
```

Comment:
Why pointer of pointer? is it better to use `commitDetail` ?

Reply:
direct use `commitDetail` will copy on `withValue` call and couldn't get modification by callled method,
use pointer of `commitDetail` can get change result, but hard to distinguish "has commitDetail" and "no commitDetail"